.eslintrc

```diff
@@ -15,6 +15,7 @@
     "no-unused-expressions": 0,
     "no-param-reassign": 0,
     "no-reserved-keys": 0,
-    "space-infix-ops": 0
+    "space-infix-ops": 0,
+    "array-bracket-spacing": [2, "never"]
```

Comment:
I'm confused, doesn't [airbnb has this rule already](https://github.com/airbnb/javascript/blob/a533e2dd35ebc05e246698e65a6e23cbd342c240/packages/eslint-config-airbnb-base/rules/style.js#L4)?